config/PropertyNames.xml

```diff
@@ -238,23 +238,6 @@ generated from the section label.
         <suffix name="MessageSizeMax" />
     </class>
 
-    <class name="deprecatedthreadpool" prefix-only="true">
```

Comment:
deprecatedthreadpool and deprecatedobjectadapter were used only by the deprecated IceBox.ServiceManager OA.